server/events/comment_parser.go

```diff
@@ -330,9 +335,21 @@ func (e *CommentParser) errMarkdown(errMsg string, command string, flagSet *pfla
 	return fmt.Sprintf("```\nError: %s.\nUsage of %s:\n%s```", errMsg, command, flagSet.FlagUsagesWrapped(usagesCols))
 }
 
-// HelpComment is the comment we add to the pull request when someone runs
-// `atlantis help`.
-var HelpComment = "```cmake\n" +
+func (e *CommentParser) HelpComment(applyDisabled bool) string {
+	buf := &bytes.Buffer{}
+	var tmpl = template.Must(template.New("").Funcs(sprig.TxtFuncMap()).Parse(helpCommentTemplate))
```

Comment:
Are you using any sprig functions?

Reply:
No sorry picked that up from an example I found online - Removed